docs/products/opensearch/howto/import-opensearch-data-using-elasticsearch-dump.rst

```diff
@@ -0,0 +1,80 @@
+
+Dump OpenSearch index using ``elasticsearch-dump``
```

Comment:
```suggestion
Backup OpenSearch indexes using ``elasticsearch-dump``
```

Reply:
Or is this for Copying OS indexes?

Reply:
I think they are all correct, so I tried to add the most close to the tool which is dumping the indexes. I wonder what the user would look first... I am not 100% sure about backup without more context. Otherwise, I'd expect the backup to be in the same system.

Reply:
dump: copy (stored data) to a different location, especially so as to protect against loss.

Reply:
I do not think is indexes is the right word because I am showing how to move one index. Here you can find similar tools or articles:

https://github.com/ianneub/elasticsearch-to-s3
https://devcoops.com/dump-es-index-to-s3/

So dump seems to fit in this case.